.travis.yml

```diff
@@ -19,3 +19,6 @@ addons:
 env:
   - CXX=g++-4.9
 sudo: false
+script:
+  - node_version=$(node -v); if [ ${node_version:1:2} = 10 ]; then yarn dtslint; fi
+  - yarn test
```

Comment:
Maybe `npm test` (and `npm run dtslint` above)? I don't think we are using yarn anywhere else? 🤔